features/release_stages.feature

```diff
@@ -1,19 +1,16 @@
 Feature: Discarding reports based on release stage
 
-Background:
-    Given I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-
     Scenario: Crash when release stage is not present in "notify release stages"
-        When I crash the app using "CrashWhenReleaseStageNotInNotifyReleaseStages"
-        And I relaunch the app
-        And I wait for 10 seconds
-        Then I should receive 0 requests
+        When I run "CrashWhenReleaseStageNotInNotifyReleaseStages" and relaunch the app
+        And I configure Bugsnag for "CrashWhenReleaseStageNotInNotifyReleaseStages"
+        And I wait for 5 seconds
```

Comment:
Is 5s long enough, when we waiting 10 before (but as before would rather we never waited for nothing to happen).